examples/LaTeX/print_check_latex.py

```diff
@@ -164,6 +164,10 @@ def main():
 
     dirac_eq.Fmt(3,r'\nabla \bm{\psi} I \sigma_{z}-e\bm{A}\bm{\psi}-m\bm{\psi}\gamma_{t} = 0')
 
+    # former bug: "value" ended up on both lines
```

Comment:
See one of the intermediate commits to see the output this used to have

Reply:
https://github.com/pygae/galgebra/pull/379/commits/7b9c5370273eb58964815d24487125466993a6a5